src/Models/Quote.php

```diff
@@ -9,6 +9,7 @@
 use Rapidez\Core\Models\Scopes\IsActiveScope;
 use TorMorten\Eventy\Facades\Eventy;
 
+// TODO: Do we still need this model?
```

Comment:
This model is still being used for the order controller, to get the order by quote hash